packages/ui/src/models/visualization/flows/camel-route-visual-entity.test.ts

```diff
@@ -366,7 +366,7 @@ describe('Camel Route', () => {
       const logWhenNode = whenNode?.getChildren()?.[0];
       expect(logWhenNode).toBeDefined();
       expect(logWhenNode!.data.path).toEqual('from.steps.1.choice.when.0.steps.0.log');
-      expect(logWhenNode!.getNodeLabel()).toEqual('log');
+      expect(logWhenNode!.getNodeLabel()).toEqual('log-1');
```

Comment:
I am baffled by this one....the test label says "should use uri as label" but here it seems we check for an id?

Reply:
mmm, suspicious, I'll check again

Reply:
ok, I see. the test description was a bit misleading

Reply:
I reworked this section to only show IDs as a fallback for entities, like `intercept*`, `errorHandler` and `onException`